lib/esp/resources/alert.rb

```diff
@@ -65,25 +69,28 @@ def self.where(clauses = {})
 
     # Find an Alert by id
     #
-    # ==== Parameter
-    #
-    # +id+ | Required | The ID of the alert to retrieve
-    #
-    # +options+ | Optional | A hash of options
-    #
-    # ===== Valid Options
-    #
-    # +include+ | The list of associated objects to return on the initial request.
+    # @overload find(id)
+    #   @param id [Integer, Numeric, #to_i] Required ID of the alert to retrieve.
+    # @overload find(id, options)
+    #   @param id [Integer, Numeric, #to_i] Required ID of the alert to retrieve.
+    #   @param options [Hash]
+    #     ===== Valid Options
     #
-    # ===== Valid Includable Associations
+    #     +include+ | The list of associated objects to return on the initial request.
     #
-    # See {API documentation}[http://api-docs.evident.io?ruby#searching-alerts] for valid arguments
+    #     ===== Valid Includable Associations
     #
-    # ==== Example
-    # alert = ESP::Alert.find(1, include: 'tags,external_account.team')
+    #     See {API documentation}[http://api-docs.evident.io?ruby#searching-alerts] for valid arguments
+    # @overload find(_non_numeric, options)
+    #   *call-seq* -> +super.all(options)+
+    #   @param _non_numeric [Object] *Example:* +:all+
```

Comment:
I think this is one of the first files I started documenting, I went off exactly how the method read. While doing another file with similar structure, I actually realized how these overloaded versions get used (with more of an emphasis on internal uses). I think I did better in some of the later files where I only included useful overloads like find(id) and find(id, options).

I'm totally onboard with trimming some of these out now that someone else agrees it's better with them streamlined.